src/main/webapp/app/view/project/admin/locations/project-location-update.component.html

```diff
@@ -0,0 +1,81 @@
+
+<div class="row justify-content-center">
+    <div class="col-8">
+        <form name="editForm" role="form" novalidate (ngSubmit)="save()" [formGroup]="editForm">
+            <h2 id="jhi-location-heading">
+                <span *ngIf="isNew; else updateTitle"
+                      jhiTranslate="eventPlannerApp.location.home.createLabel">Create a new Location</span>
+                <ng-template #updateTitle>
+                    <span jhiTranslate="eventPlannerApp.location.home.editLabel">Edit Location</span>
+                </ng-template>
+            </h2>
+
+            <div>
+                <jhi-alert-error></jhi-alert-error>
+
+                <div class="form-group" [hidden]="!editForm.get('id')!.value">
+                    <label for="id" jhiTranslate="global.field.id">ID</label>
+                    <input type="text" class="form-control" id="id" name="id" formControlName="id"
+                           readonly/>
+                </div>
+
+                <div class="form-group">
+                    <label class="form-control-label"
+                           jhiTranslate="eventPlannerApp.location.name"
+                           for="field_name">Name</label>
+                    <input type="text" class="form-control" name="name" id="field_name"
+                           formControlName="name"/>
+
+                    <div
+                        *ngIf="editForm.get('name')!.invalid && (editForm.get('name')!.dirty || editForm.get('name')!.touched)">
+                        <small class="form-text text-danger"
+                               *ngIf="editForm.get('name')?.errors?.required"
+                               jhiTranslate="entity.validation.required">
+                            This field is required.
+                        </small>
+                        <small class="form-text text-danger"
+                               *ngIf="editForm.get('name')?.errors?.maxlength"
+                               jhiTranslate="entity.validation.maxlength"
+                               [translateValues]="{ max: 50 }">
+                            This field cannot be longer than 50 characters.
+                        </small>
+                    </div>
+                </div>
+
+                <label for="field-responsibility"> Responsibility </label>
+                <ngx-autocomplete
+                    name="responsibility"
+                    id="field-responsibility"
+                    formControlName="responsibility"
+                    [options]="responsibilities"
+                    propertyAccessor="name"
+                    (autocompleteChanges)="filter($event)"
+                ></ngx-autocomplete>
+
+                <div
+                    *ngIf="
+                        editForm.get('responsibility')!.invalid &&
+                        (editForm.get('responsibility')!.dirty || editForm.get('responsibility')!.touched)
+                      "
+                >
```

Comment:
is this auto formatted? looks strange..

Reply:
Dachte ich mir auch. Aber so war es in deiner Dokumentation :)

Reply:
mag sein, aber dort verwende ich einen anderen code-style :)